src/Adapter/Presenter/Manufacturer/ManufacturerLazyArray.php

```diff
@@ -148,6 +148,7 @@ public function getImage()
      */
     public function getNbProducts()
     {
+        /* @phpstan-ignore-next-line */
```

Comment:
instead of ignoring this, you should get rid of  `|| $this->manufacturer['nb_products'] === null`
because isset already checks null value. Im pretty sure that was what phpstan was trying to tell you :smile:

Reply:
He was complaining about `strict comparison using === between mixed and null will always evaluate to false`.

So if `nb_products` will be set and will be null, isset will catch it?

Reply:
Oh yes, done.

Reply:
Yes it should.
as docs say:
`Returns <b>TRUE</b> if var exists and has value other than <b>NULL</b>, <b>FALSE</b> otherwise.`

![image](https://user-images.githubusercontent.com/31609858/222450966-55192150-df1e-40db-85e1-f78517473382.png)

Reply:
![image](https://user-images.githubusercontent.com/31609858/222451145-64ce1695-41b5-40f7-b6d3-2b06b6f98080.png)